### ISSUE TITLE
Amend Sale entities to store IPFS sale hash

### ISSUE THREAD

**Issue by adamazad** [cryptonative-ch/mesa-subgraph#49]
The IPFS hash shall points to a JSON file with essential information. An example:

```json
{
    "verified": true,
    "owner": {
        "address": "0xF006779eAbE823F8EEd05464A1628383af1f7afb",
        "label": "IDO Sale Deployer"
    },
    "name": "IDO Sale",
    "tags": [
        "IDO",
        "DAO",
        "AI"
    ],
    "links": [{
        "id": "twitter",
        "link": "https://twitter.com/idosale",
        "label": "Twitter"
    }, {
        "id": "medium",
        "link": "https://medium.com/idosale",
        "label": "Medium"
    }, {
        "id": "discord",
        "link": "https://discord.gg/idosale",
        "label": "Discord"
    }]
}
```

JSON schema

```json
{
    "$schema": "http://json-schema.org/draft-07/schema",
    "$id": "http://example.com/example.json",
    "type": "object",
    "title": "The root schema",
    "description": "The root schema comprises the entire JSON document.",
    "default": {},
    "examples": [
        {
            "verified": true,
            "owner": {
                "address": "0xF006779eAbE823F8EEd05464A1628383af1f7afb",
                "label": "IDO Sale Deployer"
            },
            "name": "IDO Sale",
            "tags": [
                "IDO",
                "DAO",
                "AI"
            ],
            "links": [
                {
                    "id": "twitter",
                    "link": "https://twitter.com/idosale",
                    "label": "Twitter"
                },
                {
                    "id": "medium",
                    "link": "https://medium.com/idosale",
                    "label": "Medium"
                },
                {
                    "id": "discord",
                    "link": "https://discord.gg/idosale",
                    "label": "Discord"
                }
            ]
        }
    ],
    "required": [
        "verified",
        "owner",
        "name",
        "tags",
        "links"
    ],
    "properties": {
        "verified": {
            "$id": "#/properties/verified",
            "type": "boolean",
            "title": "The verified schema",
            "description": "An explanation about the purpose of this instance.",
            "default": false,
            "examples": [
                true
            ]
        },
        "owner": {
            "$id": "#/properties/owner",
            "type": "object",
            "title": "The owner schema",
            "description": "An explanation about the purpose of this instance.",
            "default": {},
            "examples": [
                {
                    "address": "0xF006779eAbE823F8EEd05464A1628383af1f7afb",
                    "label": "IDO Sale Deployer"
                }
            ],
            "required": [
                "address",
                "label"
            ],
            "properties": {
                "address": {
                    "$id": "#/properties/owner/properties/address",
                    "type": "string",
                    "title": "The address schema",
                    "description": "An explanation about the purpose of this instance.",
                    "default": "",
                    "examples": [
                        "0xF006779eAbE823F8EEd05464A1628383af1f7afb"
                    ]
                },
                "label": {
                    "$id": "#/properties/owner/properties/label",
                    "type": "string",
                    "title": "The label schema",
                    "description": "An explanation about the purpose of this instance.",
                    "default": "",
                    "examples": [
                        "IDO Sale Deployer"
                    ]
                }
            },
            "additionalProperties": true
        },
        "name": {
            "$id": "#/properties/name",
            "type": "string",
            "title": "The name schema",
            "description": "An explanation about the purpose of this instance.",
            "default": "",
            "examples": [
                "IDO Sale"
            ]
        },
        "tags": {
            "$id": "#/properties/tags",
            "type": "array",
            "title": "The tags schema",
            "description": "An explanation about the purpose of this instance.",
            "default": [],
            "examples": [
                [
                    "IDO",
                    "DAO"
                ]
            ],
            "additionalItems": true,
            "items": {
                "$id": "#/properties/tags/items",
                "anyOf": [
                    {
                        "$id": "#/properties/tags/items/anyOf/0",
                        "type": "string",
                        "title": "The first anyOf schema",
                        "description": "An explanation about the purpose of this instance.",
                        "default": "",
                        "examples": [
                            "IDO",
                            "DAO"
                        ]
                    }
                ]
            }
        },
        "links": {
            "$id": "#/properties/links",
            "type": "array",
            "title": "The links schema",
            "description": "An explanation about the purpose of this instance.",
            "default": [],
            "examples": [
                [
                    {
                        "id": "twitter",
                        "link": "https://twitter.com/idosale",
                        "label": "Twitter"
                    },
                    {
                        "id": "medium",
                        "link": "https://medium.com/idosale",
                        "label": "Medium"
                    }
                ]
            ],
            "additionalItems": true,
            "items": {
                "$id": "#/properties/links/items",
                "anyOf": [
                    {
                        "$id": "#/properties/links/items/anyOf/0",
                        "type": "object",
                        "title": "The first anyOf schema",
                        "description": "An explanation about the purpose of this instance.",
                        "default": {},
                        "examples": [
                            {
                                "id": "twitter",
                                "link": "https://twitter.com/idosale",
                                "label": "Twitter"
                            }
                        ],
                        "required": [
                            "id",
                            "link",
                            "label"
                        ],
                        "properties": {
                            "id": {
                                "$id": "#/properties/links/items/anyOf/0/properties/id",
                                "type": "string",
                                "title": "The id schema",
                                "description": "An explanation about the purpose of this instance.",
                                "default": "",
                                "examples": [
                                    "twitter"
                                ]
                            },
                            "link": {
                                "$id": "#/properties/links/items/anyOf/0/properties/link",
                                "type": "string",
                                "title": "The link schema",
                                "description": "An explanation about the purpose of this instance.",
                                "default": "",
                                "examples": [
                                    "https://twitter.com/idosale"
                                ]
                            },
                            "label": {
                                "$id": "#/properties/links/items/anyOf/0/properties/label",
                                "type": "string",
                                "title": "The label schema",
                                "description": "An explanation about the purpose of this instance.",
                                "default": "",
                                "examples": [
                                    "Twitter"
                                ]
                            }
                        },
                        "additionalProperties": true
                    }
                ]
            }
        }
    },
    "additionalProperties": true
}
```

**Comment by martinkrung** [comment]
Its allready there: https://github.com/cryptonative-ch/mesa-interface/pull/240/files

Its a nice idea to use it for more info, like "verified"

**Comment by adamazad** [comment]
Implemented via #50. Closing this as vague.
